### DISCUSSION TITLE
Web based IDE

### DISCUSSION THREAD

**Discussion by saswatsamal** [hackclubiter/onEditor#12] [Resources]
Here's an article I found. May be it help us in our research.
[Link](https://dzone.com/articles/how-i-created-a-web-based-ide-in-7-days)

**Comment by saswatsamal** [comment]
Another good read. This tutorial teaches how to make an IDE using Angular.
[Link](https://medium.com/@ofir3322/create-an-online-ide-with-angular-6-nodejs-part-1-163a939a7929)

**Reply by LoopGlitch26** [reply]
Yeah, this one's great

**Comment by saswatsamal** [comment]
Have a look at Microsoft’s Monaco editor, it’s what powers VSCode under the hood and should be embedable in a webpage. 
[Monaco Editor](https://microsoft.github.io/monaco-editor/)
